hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -166,7 +176,9 @@ public void init() throws DBException {
     String table = com.yahoo.ycsb.workloads.CoreWorkload.table;
     try {
       final TableName tName = TableName.valueOf(table);
-      connection.getTable(tName).getTableDescriptor();
```

Comment:
Ah so ``getTable()`` isn't thread safe in that it could create more connection objects?